core/tauri/src/api/path.rs

```diff
@@ -491,3 +531,13 @@ pub fn log_dir(config: &Config) -> Option<PathBuf> {
 
   path
 }
+
+/// Alias of [`app_config_dir`] for backwards-compatibility purposes.
+pub fn app_dir(config: &Config) -> Option<PathBuf> {
+  app_config_dir(config)
+}
+
+/// Alias of [`app_log_dir`] for backwards-compatibility purposes.
+pub fn log_dir(config: &Config) -> Option<PathBuf> {
+  app_log_dir(config)
+}
```

Comment:
Keep the doc comments and add a deprecation note https://doc.rust-lang.org/reference/attributes/diagnostics.html?highlight=deprecate#the-deprecated-attribute saying it will be removed in v2 and suggest the new function